### PR TITLE
update addresses as per new cointype

### PR DESCRIPTION
**Pull request by alienHunterOnMars**
## 1. Overview

Update addresses for test wallets as per the new cointype 118

### CODE REVIEW

**Review by Anmol1696 — APPROVED** [review]
LGTM

File: contrib/local/configs/relayer-keys.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     {
       "name": "persistenceCore",
       "id": "core-1",
-      "hdpath": "m/44'/750'/0'/0/0",
+      "hdpath": "m/44'/118'/0'/0/0",
```
Comment (puneet2019) [inline contrib/local/configs/relayer-keys.json:27]: the relayer addresses and pubkeys below should also change

File: contrib/local/configs/keys.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     {
       "name": "test1",
       "type": "local",
-      "address": "persistence1ht0tun4u5uj4f4z83p9tncjerwu27ycsm52txm",
+      "address": "persistence152nvc6f096v6n6tr5lg50xq22ak0chsr0ru7xc",
       "pubkey": {
         "@type": "/cosmos.crypto.secp256k1.PubKey",
         "key": "AoC0SFBrAyiHrtJ+uGFuxP9CyTK8zSppylBhB+KBONNq"
```
Comment (puneet2019) [inline contrib/local/configs/keys.json:9]: the pubkeys should change

File: contrib/local/configs/validators.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
     {
       "name": "val1",
       "type": "local",
-      "address": "persistence1wh9z8dedh0mpcgzd5e2zqzsdu2j2d0zwfk3xz9",
+      "address": "persistence1fk57xyyxz89krc3nn5law2xekan84gufzagszw",
       "pubkey": {
         "@type": "/cosmos.crypto.secp256k1.PubKey",
         "key": "A3n4L7G8aFv7g3UfBZFVZA5oByH8H6E6oxBiKDpxrTPx"
```
Comment (puneet2019) [inline contrib/local/configs/validators.json:9]: the pubkeys should change for all too

**Review by puneet2019 — CHANGES_REQUESTED** [review]

**Comment by Anmol1696** [comment]
Sorry had to rever the PR. Please continute the discussion in the other PR

**Pull request merged**